images/oc-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -210,6 +210,10 @@ do
     fi
   fi
 
+  if [ "$SERVICE_TYPE" == "mongodb-single" ]; then
+    SERVICE_TYPE="mongo"
+  fi
+
```

Comment:
@shreddedbacon 
can you document why this is here? I assume that in oc-build-deploy the mongodb is called always "mongodb"? But I'm not 100% sure, so explaining this would make it a bit clearer :)